projects/canopy/src/lib/forms/radio/radio-button.component.ts

```diff
@@ -8,22 +8,39 @@ import {
   Self,
   SkipSelf,
   ViewEncapsulation,
+  Renderer2,
+  ElementRef,
 } from '@angular/core';
 import { FormGroupDirective, NgControl } from '@angular/forms';
 
 import { LgErrorStateMatcher } from '../validation/error-state-matcher';
 import { LgRadioGroupComponent } from './radio-group.component';
+import { RadioVariant } from './radio.interface';
 
 let nextUniqueId = 0;
 
 @Component({
-  selector: 'lg-radio-button',
+  selector: 'lg-radio-button, lg-filter-button',
   templateUrl: './radio-button.component.html',
-  styleUrls: ['./radio-button.component.scss'],
+  styleUrls: ['./radio-button.component.scss', './radio-button--filter.component.scss'],
   encapsulation: ViewEncapsulation.None,
 })
 export class LgRadioButtonComponent implements OnInit {
   checked = false;
+  _variant: RadioVariant;
+  set variant(variant: RadioVariant) {
```

Comment:
Similar to my question on the RadioGroup about having the variant `@Input`. Do we want the user to be able to use Filter Buttons on their own - outside of a group? If so, should we have an input for it, and should we also derive the variant from the component selector?

Reply:
I don't know if I would use it outside of a group. Maybe I would consider using a stand alone toggle or switch for only one option.I also had a talk with Michelle, when I picked up the task, whether we should also have a button variant or a stand alone filter button and the outcome was that this would be used only like this.

Reply:
good question, I think we can get rid of the `@Input` annotation as the thing in the constructor would overwrite it anyway.

This implementation will be slightly at odds with the checkbox/toggle one, so we might have to track that in an issue and refactor it at some stage.

I don't think we would use individual filter buttons, as effectively they are just individual radio options. I think in the future we may have a `pill` style component, but I think it will be semantically and visually different.